File: Daily_Questions/Strings/3110.cpp
// // https://leetcode.com/problems/score-of-a-string/description/?

// class Solution {
// public:
//     int scoreOfString(string s) {
//         int ans = 0 ;

//         for(int i = 1 ; i <s.size();i++)
//         {
//             ans = ans + abs(int(s[i]) - int(s[i-1]));
//         }

//         return ans;
//     }
// };

File: Daily_Questions/Strings/3110.java
// class Solution {

//     public int scoreOfString(String s) {
//         int ans = 0;
//         for (int i = 1; i < s.length(); i++) {
//             int currentDigit = Character.getNumericValue(s.charAt(i));
//             int previousDigit = Character.getNumericValue(s.charAt(i - 1));
//             ans += Math.abs(currentDigit - previousDigit);
//         }
//         return ans;
//     }
// }https://leetcode.com/problems/score-of-a-string/submissions/1273839783/?envType=daily-question&envId=2024-06-01